common/persistence/nosql/nosql_task_store_test.go

```diff
@@ -104,3 +109,175 @@ func TestGetTaskListSize(t *testing.T) {
 		size,
 	)
 }
+
+func getValidLeaseTaskListRequest() *persistence.LeaseTaskListRequest {
```

Comment:
Please, move all helper functions to the bottom of the file, so tests are on top.

Reply:
I think it makes it easier to read the tests when the helpers are close to where they are being used. `getValidLeaseTaskListRequest` is only used in the tests of `LeaseTaskList`. 

I suggest we move the helpers below where they are used instead of above?

Reply:
Also, if it's important whether helpers are above or below where they are used, we should have a style guide. I generally tend to put them above, because it's enforced in C.

Reply:
There is a bit around it in [Uber style guide](https://github.com/uber-go/guide/blob/master/style.md#function-grouping-and-ordering). So I suggest keeping tests on top.

Reply:
Moved to the end